src/group-constructors.js

```diff
@@ -42,6 +43,16 @@
     },
     get lastChild() {
       return this.children.length ? this.children[this.children.length - 1] : null;
+    },
+    clone: function() {
+      var clonedTiming = shared.cloneTimingInput(this._timingInput);
```

Comment:
> why do we need to clone here and above?

There used to be a fixme in the (KE) constructor saying that we should clone the input timing instead of storing it directly, so I resolved that.

As for cloning timing in clone: that was the best sense I could make of the spec. The level 1 spec [doesn't even say where to get timing from](http://w3c.github.io/web-animations/#dom-keyframeeffectreadonly-clone). The level 2 spec [says to clone it](http://w3c.github.io/web-animations/level-2/#dom-groupeffectreadonly-clone).